docs/documentation/ui-rest-api.md

```diff
@@ -172,6 +172,27 @@ Sample response:
 }
 ```
 
+### /api/v1/history/summary (GET)
+
+Returns a list of topology ID submitted by the current user.
+
+Response fields:
+
+|Field  |Value | Description|
+|---	|---	|---
+|topo-history| List| List of Topologies' IDs|
+
+Sample response:
+
+```json
+{
+    "topo-history":[
+        {"host":"wc6-1-1446571009"},
```

Comment:
I am confused if this is suppose to return a list of TopologyIDs why does the resposse have a list of map with host name as key?